src/cascadia/TerminalSettingsEditor/PercentageSignConverter.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include "PercentageSignConverter.g.h"
+#include "Utils.h"
+
+namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
+{
+    struct PercentageSignConverter : PercentageSignConverterT<PercentageSignConverter>
+    {
+        PercentageSignConverter() = default;
+
+        Windows::Foundation::IInspectable Convert(Windows::Foundation::IInspectable const& value,
+                                                  Windows::UI::Xaml::Interop::TypeName const& targetType,
+                                                  Windows::Foundation::IInspectable const& parameter,
+                                                  hstring const& language);
+
+        Windows::Foundation::IInspectable ConvertBack(Windows::Foundation::IInspectable const& value,
+                                                      Windows::UI::Xaml::Interop::TypeName const& targetType,
+                                                      Windows::Foundation::IInspectable const& parameter,
+                                                      hstring const& language);
+    };
+}
+
+namespace winrt::Microsoft::Terminal::Settings::Editor::factory_implementation
+{
+    BASIC_FACTORY(PercentageSignConverter);
+}

```

Comment:
```suggestion
DECLARE_CONVERTER(winrt::Microsoft::Terminal::Settings::Editor, PercentageSignConverter);
```

nit: here's a nice macro we have to make this easier :)

Reply:
nit: you could also throw this declaration into `PercentageConverter.h` (and the implementation into the corresponding cpp file). That way we have less files going around.

See https://github.com/microsoft/terminal/blob/main/src/cascadia/TerminalSettingsEditor/StringIsNotDesktopConverter.h, if interested. We did the same thing there.

Reply:
I'm not too much of a fan of putting that in the PercentageConverter.h file since it's not really related to this. How about (a future) refactor, we have a `DeclareConverter.h` where we can put those statements? That way, we probably could get rid of a few headers files more easily.

Reply:
My intent is to move us all away from object-based converters in the future, so we can put a pin in this.